homeassistant/components/rollershutter/zwave.py

```diff
@@ -54,6 +54,12 @@ def value_changed(self, value):
             _LOGGER.debug("Value changed on network %s", value)
 
     @property
+    def name(self):
```

Comment:
> Does this work in real life?

It does indeed work. Just tested by removing the function and re-adding the function.

> Also the linter complains that node is undefined.

That was my bad. Since I'm testing on docker with a raspberry pi, I was copy-pasting code back and forth to test. You nailed it though, it should be `self._value.node`

> use ozwcp to rename node names

Did not know this was possible (I'm new to zwave. This is my first device). Thank you for the tip.